### PR TITLE
hw4

### PR DESCRIPTION
**Pull request by qnixdev**

### CODE REVIEW

File: src/hw4/Car.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    private int fuelConsumption = 8;
+
+    public Car() {}
+
+    public Car(
+        String engine,
+        String color,
+        int tankLimit,
+        int fuelConsumption
+    ) {
```
Comment (shaitanaleksander) [inline src/hw4/Car.java:16]: bad code style 

        String engine,
         String color,
         int tankLimit,
         int fuelConsumption
         
         need to be in one line

File: src/hw4/Car.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+    public int getTankLimit() {
+        return this.tankLimit;
+    }
+
+    public void setTankLimit(int tankLimit) {
+        if (
+            tankLimit <= 0
+            || tankLimit == this.getTankLimit()
+        ) {
```
Comment (shaitanaleksander) [inline src/hw4/Car.java:59]: bad code format, make it one line

Comment (qnixdev) [reply src/hw4/Car.java]: done

File: src/hw4/ArrayComparison.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+        StringBuilder wordTwo = new StringBuilder();
+
+        for (String element : secondArray) {
+            wordTwo.append(element);
+        }
+
+        return wordOne.toString().equals(wordTwo.toString());
+    }
+}
```
[the remark this answers is not in the document]
Comment (qnixdev) [reply src/hw4/ArrayComparison.java:10]: done

File: src/hw4/Car.java (hunk 1 above)
Comment (qnixdev) [reply src/hw4/Car.java]: done

**Pull request merged**